WebRtcAec3 get "Bus Error" on ARMv7 processor

### Describe the bug

The implementation of WebRtcAec3 support contains an error (appears on ARM processors).

When using the WebRtcAec3 algorithm in a release build, you can get a "Bus Error" and the program terminates. For a debug build, this does not occur.

### Steps to reproduce

1. Build pjsip with WebRtcAec3 support for Linux ARM v7 platform (Release Build with optimizations).
2. Build your program using pjsip.
3. Run the program selecting WebRtcAec3.
4. Get "Bus Error" and program crush.

### PJSIP version

2.15

### Context

OS: Linux
CPU: ARMv7 (Little Endian)
AEC Algorithm: WebRtcAec3

### Log, call stack, etc

```shell
syslog some lines:

2024-12-09T18:18:48.152352+00:00 NanoPi-NEO-Core kernel: [358643.587283] Alignment trap: not handling instruction f9008adf at [<0064026e>]
2024-12-09T18:18:48.152474+00:00 NanoPi-NEO-Core kernel: [358643.594582] Unhandled fault: alignment exception (0xa21) at 0x00ebc824
2024-12-09T18:18:48.152495+00:00 NanoPi-NEO-Core kernel: [358643.601258] pgd = d63034c0
2024-12-09T18:18:48.162083+00:00 NanoPi-NEO-Core kernel: [358643.604155] [00ebc824] *pgd=5b269003, *pmd=54228003, *pte=e000005fe74f5f
```

`
./configure --disable-video --with-external-pa --enable-libwebrtc-aec3 --host=arm-linux-gnueabihf --target=armv7l-unknown-linux-gnueabihf --prefix=/mnt/Hell/Projects/nanopi/sysrootDeb12i/usr CC=arm-linux-gnueabihf-gcc CXX=arm-linux-gnueabihf-g++ CFLAGS="-DPJ_POOL_ALIGNMENT=8 --sysroot=/mnt/Hell/Projects/nanopi/sysrootDeb12i -mfpu=neon -march=armv7-a -DPJ_HAS_LIMITS_H=1" LDFLAGS="--sysroot=/mnt/Hell/Projects/nanopi/sysrootDeb12i"
`


Multiple warnings when compile pjsip:

`
arm-linux-gnueabihf-gcc -DPJ_AUTOCONF=1 -DPJ_POOL_ALIGNMENT=8 --sysroot=/mnt/Hell/Projects/nanopi/sysrootDeb12i -mfpu=neon -march=armv7-a -DPJ_HAS_LIMITS_H=1 -DPJ_IS_BIG_ENDIAN=0 -DPJ_IS_LITTLE_ENDIAN=1                 -I/mnt/Hell/Projects/nanopi/pjproject-2.15/pjlib/include -I/mnt/Hell/Projects/nanopi/pjproject-2.15/pjlib-util/include -I/mnt/Hell/Projects/nanopi/pjproject-2.15/pjnath/include -I/mnt/Hell/Projects/nanopi/pjproject-2.15/pjmedia/include -I/mnt/Hell/Projects/nanopi/pjproject-2.15/pjsip/include                  \
	-c -o output/sample-armv7l-unknown-linux-gnueabihf/aviplay.o \
	../src/samples/aviplay.c 
In file included from /mnt/Hell/Projects/nanopi/pjproject-2.15/pjlib/include/pj/config.h:61,
                 from /mnt/Hell/Projects/nanopi/pjproject-2.15/pjmedia/include/pjmedia/config.h:26,
                 from /mnt/Hell/Projects/nanopi/pjproject-2.15/pjmedia/include/pjmedia/types.h:27,
                 from /mnt/Hell/Projects/nanopi/pjproject-2.15/pjmedia/include/pjmedia/alaw_ulaw.h:22,
                 from /mnt/Hell/Projects/nanopi/pjproject-2.15/pjmedia/include/pjmedia.h:26,
                 from ../src/samples/aviplay.c:19:
/mnt/Hell/Projects/nanopi/pjproject-2.15/pjlib/include/pj/compat/os_auto.h:182: warning: "PJ_POOL_ALIGNMENT" redefined
  182 | #define PJ_POOL_ALIGNMENT 4
      | 
<command-line>: note: this is the location of the previous definition
`

And my recompiled program get "bus error" in the same location.

--- a/pjmedia/src/pjmedia/echo_webrtc_aec3.cpp
+++ b/pjmedia/src/pjmedia/echo_webrtc_aec3.cpp
@@ -75,8 +75,13 @@ PJ_DEF(pj_status_t) webrtc_aec3_create(pj_pool_t *pool,
     webrtc_ec *echo;
    
     *p_echo = NULL;
-    
+
+#if WEBRTC_LINUX == 1 && defined(WEBRTC_ARCH_ARM_V7)
+    /* Workaround to fix alignment trap issue on Linux ARMv7 machine. */
+    echo = new webrtc_ec();
+#else
     echo = PJ_POOL_ZALLOC_T(pool, webrtc_ec);
+#endif
     PJ_ASSERT_RETURN(echo != NULL, PJ_ENOMEM);
     
     if (clock_rate != 16000 && clock_rate != 32000 && clock_rate != 48000) {
@@ -151,6 +156,10 @@ PJ_DEF(pj_status_t) webrtc_aec3_destroy(void *state )
         delete echo->rend_buf;
         echo->rend_buf = NULL;
     }
+
+#if WEBRTC_LINUX == 1 && defined(WEBRTC_ARCH_ARM_V7)
+    delete echo;
+#endif
 
     return PJ_SUCCESS;
 }

